src/main/java/org/galatea/starter/entrypoint/IexRestController.java

```diff
@@ -48,4 +49,14 @@
     return iexService.getLastTradedPriceForSymbols(symbols);
   }
 
+  @GetMapping(value = "${mvc.iex.getHistoricalPricePath}", produces = {
```

Comment:
You should have doc for all endpoints so that it's clear what it does and what are the expect inputs/outputs